common/src/main/java/net/neoforged/gradle/common/extensions/IdeManagementExtension.java

```diff
@@ -62,7 +62,18 @@ public IdeManagementExtension(Project project) {
      *
      * @return whether this is an IntelliJ-based invocation
      */
-    public boolean isIdeaImport() {
+    public boolean isIdeaAttached() {
+        return Boolean.getBoolean("idea.active");
+    }
+
+    /**
+     * Get whether Gradle is being invoked through IntelliJ IDEA.
+     *
+     * <p>This can be through a project import, or a task execution.</p>
+     *
+     * @return whether this is an IntelliJ-based invocation
+     */
```

Comment:
This javadoc is a duplicate of the one for `isIdeaAttached`. What is the difference between the two?

Reply:
Synced is only set when the project is being loaded/reloaded within the IDE. Attached is always set to true when executing a task in an IDE.

Reply:
They are somewhat interchangeable yes. But they indicate different things.